cmd/lakectl/cmd/cat_hook_output.go

```diff
@@ -17,14 +18,14 @@ var catHookOutputCmd = &cobra.Command{
 	ValidArgsFunction: ValidArgsRepository,
 	Run: func(cmd *cobra.Command, args []string) {
 		u := MustParseRepoURI("repository", args[0])
-		Fmt("Repository: %s\n", u.String())
+		fmt.Printf("Repository: %s\n", u)
 		runID := args[1]
 		hookRunID := args[2]
 		client := getClient()
 		ctx := cmd.Context()
 		resp, err := client.GetRunHookOutputWithResponse(ctx, u.Repository, runID, hookRunID)
 		DieOnErrorOrUnexpectedStatusCode(resp, err, http.StatusOK)
-		Fmt("%s\n", string(resp.Body))
+		fmt.Printf("%s\n", string(resp.Body))
```

Comment:
```suggestion
		fmt.Println("%s", string(resp.Body))
```

Reply:
switched to use Println without %s (no format support)